### PR TITLE
F fix pan conversion rate

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #587, Giveth/giveth-dapp#2286

### CODE REVIEW

File: src/services/conversionRates/conversionRates.service.js [hunk 1]
```diff
@@ -19,29 +20,36 @@ module.exports = function conversionRates() {
...
+              enum: getValidSymbols(),
+            },
+            description: 'could be string or array of string',
+          },
+          {
+            name: 'interval',
+            in: 'query',
+            description: 'could be hourly',
+          },
+        ],
```
Comment (mohammadranjbarz) [inline src/services/conversionRates/conversionRates.service.js:52]: @aminlatifi 
it's not related to conversion rate, but I think it was good to refactor this part

**Comment by mohammadranjbarz** [comment]
@aminlatifi 
I couldn't find any solution for PAN price, when coingecko dont return price in that range time, I just added some validations for `/conversionRates` endpoint , before user could send any invalid symbol and would get response , but I restricted the user input,
and I think it's better to return `0` when we cant fetch PAN price, before we were sending `1`

File: src/utils/tokenHelper.js [hunk 2]
```diff
@@ -41,9 +41,32 @@ function getTokenBySymbol(symbol) {
   return tokensBySymbols[symbol] || { symbol };
 }
 
+const isSymbolInTokenWhitelist = symbol => {
+  return Boolean(
```
Comment (aminlatifi) [inline src/utils/tokenHelper.js:45]: Let's create a set which includes the symbols. The set should be initialized once when the application start

File: src/utils/tokenHelper.js [hunk 3]
```diff
@@ -41,9 +41,32 @@ function getTokenBySymbol(symbol) {
...
+    getWhiteListTokens().find(token => token.symbol === symbol) ||
+      // for example we dont have BTC as symbol but it is rateEqSymbol for the WBTC token in our config
+      getWhiteListTokens().find(token => token.rateEqSymbol === symbol),
+  );
+};
+
+const getValidSymbols = () => {
+  const symbols = [];
+  getWhiteListTokens().forEach(token => {
+    if (!symbols.includes(token.symbol)) {
```
Comment (aminlatifi) [inline src/utils/tokenHelper.js:55]: The same set can be used here too

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Review by aminlatifi — CHANGES_REQUESTED** [review]
See the comments

File: src/utils/tokenHelper.js (hunk 2 above)
Comment (mohammadranjbarz) [reply src/utils/tokenHelper.js]: @aminlatifi done

File: src/utils/tokenHelper.js (hunk 3 above)
Comment (mohammadranjbarz) [reply src/utils/tokenHelper.js]: @aminlatifi done

**Pull request merged**